File: C programming/Chapter-1/problem2.c
/*              PROBLEM 2
 * The length and breadth of a rectangle and radius of a circle are
 * input through the keyboard. Write a program to calculate the area
 * and perimeter of the rectangle, and the area and circumference of
 * the circle.
 */

// import the required libraries
#include <stdio.h>
#define _USE_MATH_DEFINES
#include <math.h>

int main()
{
    // declare the variables
    int l, b, r;

    // take input
    printf("Enter the length of rectangle:\n");
    scanf("%d", &l);
    printf("Enter the breadth of rectangle:\n");
    scanf("%d", &b);
    printf("Enter the radius of circle:\n");
    scanf("%d", &r);
    // do the calculations
    int rect_area = l * b;
    int rect_per = 2 * (l + b);

    float circle_area = M_PI * pow(r, 2);
    float circle_cir = 2 * M_PI * r;

    // print the input variables
    printf("length of rectangle: %d\n", l);
    printf("breadth of rectangle: %d\n", b);
    printf("Radius of the circle: %d\n", r);

    // print the results
    printf("area of rectangle: %d\n", rect_area);
    printf("perimeter of rectangle: %d\n", rect_per);
    printf("area of the circle: %f\n", circle_cir);
    printf("perimeter of the circle: %f\n", circle_area);
}

File: Java/Chapter-1/Problem2.java
/*DECLARE FLOAT length, breadth,radius, area_cir,area_rec,per_cir,per_rec
INPUT length,breadth,radius
ASSIGN area_rec to length*breadth
ASSIGN per_rec to 2*(length+breadth)
ASSIGN area_cir to 3.14*radius^2
ASSIGN per_cir to 2*3.14*radius
DISPLAY "The length of the rectangle:"
DISPLAY length
DISPLAY "The breadth of the rectangle:"
DISPLAY breadth
DISPLAY "The radius of the circle:"
DISPLAY radius
DISPLAY "The area of the rectangle:"
DISPLAY area_rec
DISPLAY "The perimeter of the rectangle:"
DISPLAY per_rec
DISPLAY "The area of the circle:"
DISPLAY area_cir
DISPLAY "The perimeter of the circle:"
DISPLAY per_cir */

//import libraries
import java.util.Scanner;

public class Problem2 {
    public static void main(String[] args) {
        // declare variables
        double length, breadth, radius, area_cir, area_rec, per_cir, per_rec;

        // take input
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter length of the rectangle:");
        length = sc.nextDouble();
        System.out.println("Enter breadth of the rectangle:");
        breadth = sc.nextDouble();
        System.out.println("Enter radius of the circle:");
        radius = sc.nextDouble();

        // calculate the results
        area_rec = length * breadth;
        per_rec = 2 * (length + breadth);
        area_cir = 3.14 * Math.pow(radius, 2);
        per_cir = 2 * 3.14 * radius;

        // display
        System.out.println("The length of the rectangle: " + length);
        System.out.println("The breadth of the rectangle: " + breadth);
        System.out.println("The radius of the circle: " + radius);
        System.out.println("The area of the rectangle: " + area_rec);
        System.out.println("The perimeter of the rectangle: " + per_rec);
        System.out.println("The area of the circle: " + area_cir);
        System.out.println("The perimeter of the circle: " + per_cir);

    }
}
